Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -6746,12 +6756,62 @@ bool IWindowExplorer.TestElementLocators(ElementInfo EI, bool GetOutAfterFoundEl
                 }
             }
             finally
+            {
+                Driver.Manage().Timeouts().ImplicitWait = (TimeSpan.FromSeconds((int)ImplicitWait));
+            }
+        }
+
+
+        void IWindowExplorer.CollectOriginalElementsDataForDeltaCheck(ObservableList<ElementInfo> mOriginalList)
+        {
+            try
+            {
+                mIsDriverBusy = true;
+                Driver.Manage().Timeouts().ImplicitWait = new TimeSpan(0, 0, 0);
+
+                foreach (ElementInfo EI in mOriginalList)
+                {
+                    EI.ElementStatus = ElementInfo.eElementStatus.Pending;
+                }
+
+
+                foreach (ElementInfo EI in mOriginalList)
+                {
+                    try
+                    {
+                        SwitchFrame(EI);
+                        IWebElement e = LocateElementByLocators(EI.Locators);
+                        if (e != null)
+                        {
+                            EI.ElementObject = e;
+                            EI.ElementStatus = ElementInfo.eElementStatus.Passed;
+                        }
+                        else
```

Comment:
don't mess the DeltaStatus setting here and in Wizard, this function only need to reset all values and pull you needed values to compare by, in Wizard you will have all DeltaStatus setting logic instead in multiple places

Reply:
???